default/stringtables/en.txt

```diff
@@ -6238,6 +6241,40 @@ Opinions of Empires:
 OPINIONS_OF_OTHER_SPECIES
 Opinions of Other Species:
 
+# Species Pedia Categories
+
+LITHIC_SPECIES_CLASS
+Lithic Species
+
+LITHIC_SPECIES_CLASS_DESC
+List of [[encyclopedia LITHIC_SPECIES_TITLE]] species:
+
+ORGANIC_SPECIES_CLASS
+Organic Species
+
+ORGANIC_SPECIES_CLASS_DESC
+List of [[encyclopedia ORGANIC_SPECIES_TITLE]] species:
+
+PHOTOTROPHIC_SPECIES_CLASS
+Phototrophic Species
+
+PHOTOTROPHIC_SPECIES_CLASS_DESC
+List of [[encyclopedia PHOTOTROPHIC_SPECIES_TITLE]] species:
+
+ROBOTIC_SPECIES_CLASS
+Robotic Species
+
+ROBOTIC_SPECIES_CLASS_DESC
+List of [[encyclopedia ROBOTIC_SPECIES_TITLE]] species:
+
+SELF_SUSTAINING_SPECIES_CLASS
```

Comment:
Actually, it seems the blank value is rendered in-game as a single space (the undertitle line is a little bit shifted to the right -from one space, almost unnoticeable).

No error in the log, AFAIK.